exchanges/stream/buffer/buffer_types.go

```diff
@@ -23,9 +27,16 @@ type Orderbook struct {
 	m                     sync.Mutex
 }
 
+// orderbookHolder defines a store of pending updates and a pointer to the
+// orderbook depth
 type orderbookHolder struct {
-	ob     *orderbook.Base
+	ob     *orderbook.Depth
 	buffer *[]Update
+	// Reduces the amount of outbound alerts to the data handler for example
+	// coinbasepro can have up too 100 updates per second introducing overhead.
+	// The sync agent only requires an alert every 15 seconds for a specific
+	// currency.
+	ticker *time.Ticker
```

Comment:
Can this functionality be overridable for people who want all the datas displayed?

Reply:
I guess it could link up to config; but why you would want to spam your file or stdout? Is it checking throughput on implementation?

Reply:
For the same reason we have verbose and debug logs

Reply:
Added but not linked up to config, for now, I have just linked it up to the verbose bool.

Reply:
Thanks! I think that's good enough 🎉